src/types.ts

```diff
@@ -62,12 +61,9 @@ export type DeskproUser = {
 };
 
 export type ContextData = {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    app: any,
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    currentAgent: any,
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    env: any,
+    app: object;
+    currentAgent: object,
+    env: object,
     user: DeskproUser,
```

Comment:
This should be strongly typed, we know the format of this so we should declare it ahead of time.

Reply:
deleted `app`, `env` and `currentAgent ` because we aren't using them.